test/unit/proc_macro/proc_macro_2015.rs

```diff
@@ -0,0 +1,10 @@
+extern crate proc_macro;
+
+use proc_macro::TokenStream;
+
+/// This macro is a no-op; it is exceedingly simple as a result
```

Comment:
Suggestion: I think the comment is not necessary. If you want to make the code more intention revealing, name the macro as Noop?

Reply:
Please apply the same change below too.

Reply:
This was lifted from [the existing examples](https://github.com/bazelbuild/rules_rust/tree/19c74ce6ad55ef07bfeb36af1889c46da36d5a07/examples/proc_macro) should those be updated too?

Reply:
I've refactored the code to use code [straight from the book](https://doc.rust-lang.org/reference/procedural-macros.html#function-like-procedural-macros)